packages/apollo-codegen-swift/src/language.ts

```diff
@@ -62,7 +62,15 @@ export class SwiftGenerator<Context> extends CodeGenerator<
   }
 
   multilineString(string: string) {
-    this.printOnNewline(`"${escapedString(string)}"`);
+    this.printNewline();
+    this.print(`"""`);
+
+    string.split("\n").forEach(line => {
+      this.printNewline();
+      this.print(line);
+    });
+    this.printNewline();
+    this.print(`"""`);
```

Comment:
Can we indent the multiline string by the appropriate amount? The indentation of the final `"""` is trimmed from all lines (which must have at least the same indentation or it's a syntax error), so indenting this won't affect the runtime string value.

Reply:
Yeah fair point - I'll take a better look at this today or tomorrow